lib/timeline/component/ItemSet.js

```diff
@@ -541,6 +541,7 @@ ItemSet.prototype.show = function() {
 
 /**
  * Activates the popup timer to show the given popup after a fixed time.
+ * @param {any} popup
```

Comment:
:+1:

Reply:
cherry-picked this into #4256 to provide a more logical separation of concern (it's a bit out of place  in this PR)